fluent-plugin-events/test/plugin/test_in_events.rb

```diff
@@ -186,4 +186,65 @@ def connect_kubernetes_with_api_version(driver)
     
     driver.start_monitor
   end
+
+  sub_test_case 'exception from kubeclient will not interrupt execution of start monitor' do
+    test 'exception from kubeclient call in create_config_map' do
+      config = %([])
+      driver = create_driver(config).instance
+      connect_kubernetes_with_api_version(driver)
+      driver.instance_variable_set(:@clients, @clients)
+      driver.instance_variable_set(:@last_recreated, 0)
```

Comment:
is this needed in test cases that don't call `start_monitor`? Also it looks like these 5 lines are being repeated in almost every test case, maybe we need to DRY or need something similar to the `beforeEach()` in scala

Reply:
Good point. Going to refactor this.